README.md

```diff
@@ -17,6 +17,7 @@ Slick-pg
 - Inet/MacAddr
 - `text` Search
 - `postgis` Geometry
+- Large Object
```

Comment:
I'd like to put it under 'Currently supported pg features' instead of here, because when we said it's a pg data type, we mean its corresponding scala type and type related operator/functions.